python/nammu/controller/NammuController.py

```diff
@@ -618,24 +619,43 @@ def process_validation_errors(self, oracc_log):
         validation_errors_server = {}
         for line in oracc_log.splitlines():
             if ':' in line:
-                line_number = line.split(':')[1]
+                try:
+                    server_filename = line.split(':')[0]
+                except IndexError:
+                    continue
+                try:
+                    line_number = line.split(':')[1]
+                except IndexError:
+                    continue
                 try:
                     project_id = line.split(':')[2]
                 except IndexError:
                     continue
-                error_message = line.split(project_id + ':')[1]
+                try:
+                    error_message = line.split(project_id + ':')[1]
+                except IndexError:
+                    continue
```

Comment:
If all these try/except are handling the exceptions the same way, you could group them in one single try/except. That'd be better at least for readability.